### PR TITLE
Add ScriptBlockText to Winlogbeat Configs

### PR DESCRIPTION
**Pull request by JohnConnorRF**
Added ScriptBlockText as a field for winlogbeat configs as per https://www.elastic.co/guide/en/beats/winlogbeat/master/exported-fields-winlog.html. 

This enables using Sigma queries for ScriptBlock Logging in ElasticSearch SIEM instances.

### CODE REVIEW

File: tools/config/winlogbeat-modules-enabled.yml [hunk 1]
```diff
@@ -135,6 +135,7 @@ fieldmappings:
   Product: winlog.event_data.Product
   Properties: winlog.event_data.Properties
   RuleName: winlog.event_data.RuleName
+  ScriptBlockText: winlog.event_data.ScriptBlockText
```
Comment (neu5ron) [inline tools/config/winlogbeat-modules-enabled.yml:138]: modules enabled = ECS.
should be powershell.file.script_block_text
see: https://github.com/elastic/beats/blob/048c3cc19bf43c8a6b332afaafdd0a2eb8e5bd49/x-pack/winlogbeat/module/powershell/config/winlogbeat-powershell.js#L171-L178

**Review by neu5ron — COMMENTED** [review]
modules enabled = ECS.
should be powershell.file.script_block_text
see: https://github.com/elastic/beats/blob/048c3cc19bf43c8a6b332afaafdd0a2eb8e5bd49/x-pack/winlogbeat/module/powershell/config/winlogbeat-powershell.js#L171-L178

File: tools/config/winlogbeat-modules-enabled.yml (hunk 1 above)
Comment (JohnConnorRF) [reply tools/config/winlogbeat-modules-enabled.yml]: Great catch! It's been updated accordingly.

**Comment by Neo23x0** [comment]
Is this ready to be merged?

**Comment by neu5ron** [comment]
@Neo23x0 LGTM

**Pull request merged**